modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -2682,6 +2682,76 @@ public void testPartialUpdateObjectEntry() throws Exception {
 			});
 	}
 
+	@Test
+	public void testSearchObjectEntriesWhenEntryTitleFieldIsExternalReferenceCode()
+		throws Exception {
+
+		ObjectDefinition childObjectDefinition = _createObjectDefinition(
+			Arrays.asList(
+				new TextObjectFieldBuilder(
+				).labelMap(
+					LocalizedMapUtil.getLocalizedMap(
+						RandomTestUtil.randomString())
+				).name(
+					"textObjectFieldName"
+				).build()));
+
+		List<ObjectField> objectFields =
+			objectFieldLocalService.getObjectFields(
+				_objectDefinition3.getObjectDefinitionId());
+
+		ObjectField externalReferenceCodeObjectField = null;
+
+		for (ObjectField objectField : objectFields) {
+			if (StringUtil.equals(
+					objectField.getName(), "externalReferenceCode")) {
+
+				externalReferenceCodeObjectField = objectField;
+
+				break;
+			}
+		}
```

Comment:
You can get the externalReferenceCode objectField this way:
```java 
   		ObjectField objectField = objectFieldLocalService.getObjectField(
			_objectDefinition3.getObjectDefinitionId(), "externalReferenceCode");
```

Reply:
Hi @p-albuquerque , I made this change on [a5cb0a952bf2957cec569d843d62e4f151f81d54](https://github.com/liferay-objects/liferay-portal/pull/3081/commits/a5cb0a952bf2957cec569d843d62e4f151f81d54).
Thanks